netbox/dcim/tests/test_models.py

```diff
@@ -549,24 +604,27 @@ def test_direct_connection(self):
         self.assertIsNone(endpoint_a.connection_status)
         self.assertIsNone(endpoint_b.connection_status)
 
-    def test_connection_via_single_rear_port(self):
```

Comment:
The test in `test_connection_via_single_rear_port` should fail because Panel 1's rear port has multiple positions. This is already testen in `test_singlepos_rearport_connections` and `test_multipos_rearport_connections` above.

Reply:
The fact that this test exists implies that this is a supported topology. We can't simply change it without explicit documentation that this connection is no longer going to be supported.

Reply:
This will fail on this test because Panel 1 has 4 positions:
https://github.com/netbox-community/netbox/blob/develop/netbox/dcim/models/__init__.py#L2199-L2200

But this test never caught it because `full_clean()` was never called.